Algorithm.Framework/Alphas/EmaCrossAlphaModel.cs

```diff
@@ -104,6 +104,8 @@ public override void OnSecuritiesChanged(QCAlgorithm algorithm, SecurityChanges
                     // create fast/slow EMAs
                     var fast = algorithm.EMA(added.Symbol, _fastPeriod, _resolution);
                     var slow = algorithm.EMA(added.Symbol, _slowPeriod, _resolution);
```

Comment:
The indicators need to be created by their constructor and registered to a consolidator, so we can remove the subscription to the consolidator when the security is removed.